README.md

```diff
@@ -23,6 +23,8 @@ These are the requirements which motivated the conventions framework.
 - Conventions must not require approval of a central authority (e.g. Zarr Steering Council, Zarr Implementation Council); anyone can create a Convention
 - Conventions should be self describing in terms of the specific metadata fields they contain.
 - Convention metadata attributes must not be able to collide or conflict with metadata attributes of other conventions.
```

Comment:
```suggestion
```
This is no longer a requirement, since the revised structure allows two conventions to use the same root key.